bin/plugin/commands/readme.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+const config = require( '../config' );
+const { getModuleDescriptions } = require( './common' );
+const { getChangelog } = require( './changelog' );
+
+/**
+ * @typedef WPReadmeCommandOptions
+ *
+ * @property {string=} directory Optional directory, default is the root `/modules` directory.
+ * @property {string=} milestone Optional milestone title, to update the changelog in the readme.
+ * @property {string=} token     Optional personal GitHub access token, only relevant for changelog updates.
+ */
+
+/**
+ * @typedef WPReadmeSettings
+ *
+ * @property {string}  owner     GitHub repository owner.
+ * @property {string}  repo      GitHub repository name.
+ * @property {string}  directory Modules directory.
+ * @property {string=} milestone Optional milestone title, to update the changelog in the readme.
+ * @property {string=} token     Optional personal GitHub access token, only relevant for changelog updates.
+ */
+
+exports.options = [
+	{
+		argname: '-d, --directory <directory>',
+		description: 'Modules directory',
+	},
+	{
+		argname: '-m, --milestone <milestone>',
+		description: 'Milestone title, to update the changelog',
+	},
+	{
+		argname: '-t, --token <token>',
+		description: 'GitHub token',
+	},
+];
+
+/**
+ * Command that updates the `readme.txt` file.
+ *
+ * @param {WPReadmeCommandOptions} opt
+ */
+exports.handler = async ( opt ) => {
+	await updateReadme( {
+		owner: config.githubRepositoryOwner,
+		repo: config.githubRepositoryName,
+		directory: opt.directory || 'modules',
+		milestone: opt.milestone,
+		token: opt.token,
+	} );
+};
+
+/**
+ * Returns a promise resolving to the module description list string for the `readme.txt` file.
+ *
+ * @param {WPReadmeSettings} settings Readme settings.
+ *
+ * @return {Promise<string>} Promise resolving to module description list in markdown, with trailing newline.
+ */
+async function getModuleDescriptionList( settings ) {
+	const moduleDescriptions = await getModuleDescriptions(
+		settings.directory
+	);
+
+	return moduleDescriptions
+		.map( ( moduleData ) => {
+			return `* **${ moduleData.name }:** ${ moduleData.description }`;
+		} )
```

Comment:
The `return` can be removed from this statement.

```suggestion
		.map( ( moduleData ) => ( `* **${ moduleData.name }:** ${ moduleData.description }` ) )
```